tests/layer_tests/common/tflite_layer_test_class.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+import tensorflow as tf
+from tensorflow.lite.tools import flatbuffer_utils as utils
+from common.layer_test_class import CommonLayerTest
+from common.utils.tflite_utils import get_tflite_results, get_tensors_from_graph
+
+class TFLiteLayerTest(CommonLayerTest):
+    model_path = None
+    inputs = None
+    outputs = None
+    allowed_ops = None
+
+    def make_model(self, params):
+        raise RuntimeError("This is Tensorflow Lite base layer test class, "
+                           "please implement make_model function for the specific test")
+
+    def produce_tflite_model(self, framework_model, save_path):
+        with tf.Graph().as_default() as g:
+            tf.graph_util.import_graph_def(framework_model, name="")
+            input_tensors = get_tensors_from_graph(g, self.inputs)
+            output_tensors = get_tensors_from_graph(g, self.outputs)
+
+        tflite_model = tf.compat.v1.lite.TFLiteConverter(framework_model,
+                                                         input_tensors=input_tensors,
+                                                         output_tensors=output_tensors).convert()
```

Comment:
May we have TF model that is not converted to TF Lite model? Is any restriction here?

Reply:
Yes, there are certain restrictions. Please see on the tensorflow.org